apps/nextjs-website/react-components/types/PressRelease/PressRelease.types.ts

```diff
@@ -0,0 +1,9 @@
+export interface PressReleaseProps {
+  title: string;
+  body: JSX.Element;
+  eyelet: string;
```

Comment:
```suggestion
  eyelet?: string;
```
From everything in src/app, eyelet should be optional.